### ISSUE TITLE
Generate basic projects without a spec file.

### ISSUE THREAD

**Issue by pepibumur** [yonaskolb/XcodeGen#18]
I just had an idea. What do you think about being able to generate simple projects from the command line with XcodeGen. Something like:

```bash
xcodegen init -platform iOS -name MyTest --no-tests
```

That would create the `xcodegen.yml` file and generate the project from it. How does that sound?

**Comment by yonaskolb** [comment]
Yeah, that's a good idea. I guess it would fulfil a similar purpose to https://github.com/JohnSundell/Xgen but we'd have more power underneath for some extra flexibility and options

**Comment by pepibumur** [comment]
🎉  genau!
